modules/ding_sections/modules/ding_sections_term_panel/ding_sections_term_panel.module

```diff
@@ -262,93 +262,24 @@ function ding_sections_term_panel_create_term_variant($term) {
   $variant_title = t('Sections term (@term)', array('@term' => $term->name));
   $pipeline = module_exists('panels_ipe') ? 'ipe' : 'standard';
 
-  $base_handler = page_manager_load_task_handler('term_view', '', 'term_section_panel_context');
+  $export = page_manager_export_task_handler(page_manager_load_task_handler('term_view', '', 'term_section_panel_context'));
```

Comment:
This "hack" with eval seams like something that needs an inline comment! 

So this loads an pane/panels as an string encode config (like seen in features exports) an creates an new variant of that pane/panel?